diff --git a/node_modules/react-native-paper/lib/module/components/MaterialCommunityIcon.js b/node_modules/react-native-paper/lib/module/components/MaterialCommunityIcon.js
index 472fee1..c5ef161 100644
--- a/node_modules/react-native-paper/lib/module/components/MaterialCommunityIcon.js
+++ b/node_modules/react-native-paper/lib/module/components/MaterialCommunityIcon.js
@@ -15,19 +15,16 @@ export const accessibilityProps = Platform.OS === 'web' ? {
  */
 const loadIconModule = () => {
   try {
-    return require('@react-native-vector-icons/material-design-icons').default;
+    return require('@expo/vector-icons/MaterialCommunityIcons').default;
   } catch (e) {
     try {
-      return require('@expo/vector-icons/MaterialCommunityIcons').default;
+      return require('react-native-vector-icons/MaterialCommunityIcons').default;
     } catch (e) {
-      try {
-        return require('react-native-vector-icons/MaterialCommunityIcons').default;
-      } catch (e) {
-        return null;
-      }
+      return null;
     }
   }
 };
+
 const IconModule = loadIconModule();
 
 /**
